### ISSUE TITLE
Implementing Serde Traits?

### ISSUE THREAD

**Issue by holtgrewe** [rust-bio/rust-bio-types#13]
Would it make sense to implement `serde` traits?

**Comment by pmarks** [comment]
yes - definitely. Some types already do (https://github.com/rust-bio/rust-bio-types/blob/master/src/alignment.rs#L18).

We should expose them via a feature flag so that people who don't need serde aren't forced to have it.

**Comment by holtgrewe** [comment]
@pmarks thanks, cf. #14 splits out serde support into a feature (that is by default not enabled)

**Comment by holtgrewe** [comment]
I'm really uncertain whether there should be separate tests for the serde feature... I added running tests with all features enabled, though, which means that the create is compiled with and without serde support, I think.
